### PR TITLE
Add mkrf_ru spider (~51000 locations).

### PR DESCRIPTION
**Pull request by panasonic-mafia**
Open data from Ministry of Culture of Russia, updated pretty often.

### CODE REVIEW

**Comment by iandees** [comment]
Is the Ministry of Culture an official source or just official-sounding?

**Comment by panasonic-mafia** [comment]
@iandees Ministry of Culture is an official source. This is their open-data website on which they publish and maintain all datasets.

**Comment by iandees** [comment]
Got it, thanks.

File: locations/spiders/mkrf_ru.py [hunk 1]
```diff
@@ -0,0 +1,155 @@
...
+        "use:commercial": "permit",
+    }
+    # TODO: add more datasets from https://opendata.mkrf.ru/item/api
+    datasets = ["museums", "cinema", "libraries", "culture_palaces_clubs"]
+
+    api_key = "be088ddb94bfd718a196c7ac7f67d32303ba69681948ec0a21744cdd4f78bd16"
+
+    def start_requests(self):
+        for dataset in self.datasets:
+            yield Request(
```
Comment (Cj-Malone) [inline locations/spiders/mkrf_ru.py:86]: FYI JsonRequest will add the Content-Type header automatically

**Pull request merged**